runtime/src/main/java/org/corfudb/protocols/wireprotocol/DataType.java

```diff
@@ -20,11 +20,12 @@
 
     final int val;
 
+    public static final Map<Byte, DataType> typeMap =
```

Comment:
are we just moving this code?

Reply:
yeah..trying to reduce the amount of issues from codacy